swagger/v3/api_spec.json

```diff
@@ -4439,6 +4439,11 @@
             "description": "Return a school with the specified Unique Reference Number (URN).",
             "type": "string",
             "example": "106286"
+          },
+          "updated_since": {
```

Comment:
could you add `update_since` to the example parameters [here](https://ecf-review-pr-3411.london.cloudapps.digital/api-reference/reference-v3.html#api-v3-schools-ecf-get-parameters)? same for statements?

Reply:
Added 👍